src/player.h

```diff
@@ -1134,6 +1134,7 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
 
         // Prints message(s) about current health
         void print_health() const;
+        bool reading_from_inventory= false;
```

Comment:
If you really need this information, store it in the `player_activity` (e.g. as entry in the `values` vector, like `activity_on_turn_move_items` uses it). That will ensure it's properly stored/loaded in the saves (which your current implement lacks).

The `player` class is already quite complex, another member that only has a meaning when a specific type of activity is performed just makes it worse.